package.json

```diff
@@ -69,5 +69,7 @@
     "webpack": "^5.36.2",
     "webpack-cli": "^4.7.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "bootstrap": "^5.1.0"
```

Comment:
Can you please remove this? I don't know why it's been added in the first place, there really is no need for having bootstrap in the project.